tests/behat/features/content_type.civictheme_alert.view.feature

```diff
@@ -118,3 +124,35 @@ Feature: CivicTheme Alert content type render
     And I wait for AJAX to finish
     Then I should see the text "[TEST] Test alert body all pages"
     And I should not see the text "[TEST] Test alert body all pages future"
+
+  @api @javascript
+  Scenario: Alerts visibility
+    Given I am logged in as a user with the "Site Administrator" role
+    When I edit civictheme_alert "[TEST] Test alert title visibility"
+    And I fill in "Page visibility" with:
+      """
```

Comment:
Added test to check for functionality of pages and not the "ensure it does not have html in it" check api endpoint response 

as it needs https://github.com/ubirak/rest-api-behat-extension to be configured and added to behat setup